hangulime.js

```diff
@@ -363,7 +363,7 @@ var jamo_med_o_o    = new Jamo(jamo_med_o,   "hh", JAMO_MEDIAL, 'ᆂ', undefined
 var jamo_med_o_u    = new Jamo(jamo_med_o,   "hn", JAMO_MEDIAL, 'ᆃ', undefined);
 var jamo_med_o_ya   = new Jamo(jamo_med_o,   "hi", JAMO_MEDIAL, 'ᆦ', undefined);
 var jamo_med_o_yae  = new Jamo(jamo_med_o,   "hO", JAMO_MEDIAL, 'ᆧ', undefined);
-var jamo_med_yo_ya  = new Jamo(jamo_med_yo,  "yi", JAMO_MEDIAL, 'ᆄ', undefined);
+var jamo_med_yo_yeo  = new Jamo(jamo_med_yo,  "yi", JAMO_MEDIAL, 'ᆄ', undefined);
```

Comment:
issue(blocking): This is wrong, please revert.

Reply:
I don't think this is wrong. It has to do with how the word is displayed in MacOS natvie fonts. You see, in MacOS native fonts, U+1184 & U+1186 both look the exact same. However, if you type them out in the IME, you will find out that jamo_med_yo_ya and jamo_med_yo_yeo display the correct versions.

Reply:
Then the MacOS native Hangul font must be defective. This IME is only concerned with inputting the correct and expected Hangul characters and should not account for any defects in Hangul fonts, as [U+1184 is clearly defined as a combination of YO ᅭ and YA ᅣ in Unicode](https://util.unicode.org/UnicodeJsps/character.jsp?a=1184) and not anything else.

In any case, your change now means that `jamo_med_yo_yeo` is defined twice (see lines 366 and 368) and `jamo_med_yo_ya` doesn't exist anymore, which results in an error that prevents the IME from working:

```
Uncaught ReferenceError: jamo_med_yo_ya is not defined
```

Reply:
I see. I thought that error was fixed. I pushed my results now, should be fine.